### PR TITLE
first commit

### PR DESCRIPTION
**Pull request by d-beschastnyi**
https://d-beschastnyi.github.io/layout_moyo-header/
https://d-beschastnyi.github.io/layout_moyo-header/report/html_report/

### CODE REVIEW

**Review by qoa11a — CHANGES_REQUESTED** [review]
image
All the tests are failed. Let's work on it!

File: src/index.html [hunk 1]
```diff
@@ -9,6 +9,52 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <div class="wrapper">
+        <div class="header__logo">
+          <img src="images/logo.png" alt="MOYO logo">
+        </div>
```
Comment (DarkMistyRoom) [inline src/index.html:16]: What about this requirement?
image

File: src/index.html [hunk 2]
```diff
@@ -9,6 +9,52 @@
...
+            <li class="navigation__item">
+              <a href="#" class="navigation__link">Smartphones</a>
+            </li>
+
+            <li class="navigation__item">
+              <a
+                href="#"
+                class="navigation__link"
+                data-qa="hover">Laptops & Computers
+              </a>
```
Comment (DarkMistyRoom) [inline src/index.html:37]:
```suggestion
              <a
                href="#"
                class="navigation__link"
                data-qa="hover"
              >
                Laptops & Computers
              </a>
```

File: src/style.css [hunk 3]
```diff
@@ -1 +1,63 @@
-/* Styles go here */
+@import url("https://fonts.googleapis.com/css?family=Roboto:500&display=swap");
+
+* {
+  margin: 0;
+  padding: 0;
+}
```
Comment (DarkMistyRoom) [inline src/style.css:6]: Don't use * selector for zeroing out your margins or paddings. It's still inefficient for browser to read your web document

File: src/style.css [hunk 4]
```diff
@@ -1 +1,63 @@
...
+body {
+  font-family: "Roboto", sans-serif;
+}
+
+.wrapper {
+  margin: 0 50px;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
```
Comment (DarkMistyRoom) [inline src/style.css:17]: Ok, but you should know that you can just set paddings to the header

File: src/style.css [hunk 5]
```diff
@@ -1 +1,63 @@
...
+
+.header__logo {
+  height: 40px;
+}
+
+.navigation__list {
+
+  list-style: none;
```
Comment (DarkMistyRoom) [inline src/style.css:25]:
```suggestion
.navigation__list {
  list-style: none;
```

**Review by DarkMistyRoom — CHANGES_REQUESTED** [review]

**Review by AlisaVlasova — APPROVED** [review]

**Pull request closed without merging**